rmgpy/ml/estimator.py

```diff
@@ -1,6 +1,3 @@
-#!/usr/bin/env python
```

Comment:
I might have things mixed up, but isn't there a difference between unicode and UTF-8? I thought that unicode was just a matching of characters to unique indexes (codepoints) and that UTF-8 was one of a few different ways of encoding unicode characters (for example there is also UTF-16 and others). In fact, it appears that python behind the scenes is not using UTF-8 for it's encoding because it needs every character to take up the same number of bytes to be able to do random access to individual characters in strings. (see https://rushter.com/blog/python-strings-and-memory/).

That being said, UTF-8 is almost ubiquitous, so it is unlikely that someone would have trouble with this (I think most OSs use UTF-8 by default and if not most things are good about being able to automatically figure out encodings when they are standard like UTF-8), but I don't think we need to get rid of these statements either.

My interpretation of this could be wrong though, so please correct me if so.

Reply:
Does this just use the system default encoding? Or does python set this as the default regardless of the OS? Either way I am almost certain that UTF-8 is the default encoding for mac and Linux, so we are fine.

Again, I don't think this is an issue, just curious to know.

Reply:
I think you're mixing up things. The UTF-8 magic comment simply tells the Python interpreter to assume the file is in UTF-8, which it now does by default in Python 3. That doesn't actually mean the file is UTF-8 which depends on how the editor you are using encodes the file, so it's up to the programmer to ensure that their editor is set up correctly.

Reply:
As @amarkpayne and I just discussed, it could still be useful to keep the comment in if editors rely on reading it to determine encoding. However, PyCharm, vim, emacs, probably many others don't need it.